cli/cliui/select.go

```diff
@@ -75,33 +75,172 @@ func Select(inv *serpent.Invocation, opts SelectOptions) (string, error) {
 		return opts.Options[0], nil
 	}
 
-	var defaultOption interface{}
-	if opts.Default != "" {
-		defaultOption = opts.Default
+	initialModel := selectModel{
+		search:     textinput.New(),
+		hideSearch: opts.HideSearch,
+		options:    opts.Options,
+		height:     opts.Size,
+		message:    opts.Message,
 	}
 
+	if initialModel.height == 0 {
+		initialModel.height = 7
```

Comment:
nit: Is there any significance to this? Would it make sense to extract to a `var`?

Reply:
The 7 comes from the previous default in the survey library https://github.com/AlecAivazis/survey/blob/160123ed7175f77bc8625b59c44b8dc03f5333e3/survey.go#L24

Maybe have a `const defaultModelHeight = 7` to make it less magic?